src/main/java/com/synopsys/integration/detect/configuration/DetectableOptionFactory.java

```diff
@@ -87,7 +98,13 @@ public ClangDetectableOptions createClangDetectableOptions() {
     }
 
     public ComposerLockDetectableOptions createComposerLockDetectableOptions() {
-        Boolean includedDevDependencies = getValue(DetectProperties.DETECT_PACKAGIST_INCLUDE_DEV_DEPENDENCIES);
+        boolean includedDevDependencies = Boolean.TRUE.equals(getValue(DetectProperties.DETECT_PACKAGIST_INCLUDE_DEV_DEPENDENCIES));
+        if (detectConfiguration.wasPropertyProvided(DetectProperties.DETECT_PACKAGIST_DEPENDENCY_TYPES_EXCLUDED.getProperty())) {
+            List<PackagistDependencyType> excludedDependencyTypes = PropertyConfigUtils.getNoneList(detectConfiguration, DetectProperties.DETECT_PACKAGIST_DEPENDENCY_TYPES_EXCLUDED.getProperty()).representedValues();
```

Comment:
I'd really prefer if this were a generic utility method. At least the creation of the filter. You could get even slightly fancier for properties that only have a single value at present.